modules/appeals_api/config/schemas/v1/10182_headers.json

```diff
@@ -71,6 +70,39 @@
       ]
     },
 
+    "X-VA-Claimant-First-Name": {
+      "allOf": [
+        { "description": "first name" },
```

Comment:
Maybe "Non Veteran claimant's first name."

Reply:
Given the confusion that can come from a Veteran being the claimant, and then also asking for claimant info, it may be nice to clear up any misunderstandings through the documentation.

Reply:
Good call! Thank you!